<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wvdial</class>
 <widget class="QMainWindow" name="wvdial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wvdial-gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QTextEdit" name="statusOutPut"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pbConnect">
      <property name="text">
       <string>&amp;Connect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pbQuit">
      <property name="text">
       <string>&amp;Quit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pbClear">
      <property name="text">
       <string>&amp;Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
